src/sections/upload-dataset-files/FileUploader.module.scss

```diff
@@ -9,10 +9,36 @@
     grid-template-columns: repeat(4, auto);
 }
 
+.uploaded_files {
+    display: grid;
+    grid-template-columns: 4rem auto auto auto 3.5rem;
+}
+
+.selected_file>div {
+    margin-bottom: 0.5rem;
+    background-color: #ffc !important;
+}
+
+.file:nth-of-type(even)>div {
+    margin-bottom: 0.5rem;
+    padding: 0.5rem;
+    background-color: #f8f9fc;
+}
+
+.file:nth-of-type(odd)>div {
+    margin-bottom: 0.5rem;
+    padding: 0.5rem;
+    background-color: white;
+}
+
 .file {
     display: contents;
 }
 
+.selected_file {
+    display: contents;
```

Comment:
What is the need of using `display: contents`?

Reply:
If not, it looks like this:
![image](https://github.com/user-attachments/assets/48f21057-a06c-489c-992b-0a92b4ce7c83)

Reply:
I see, this is what display: contents do:

> display: contents causes an element's children to appear as if they were direct children of the element's parent, ignoring the element itself. This can be useful when a wrapper element should be ignored when using CSS grid or similar layout techniques.

I think it's a bit fragile css hack and we should use a table here perhaps and not a `Card` to render the list of uploaded files. But anyways, looks like something that we could refactor in another iteration, but good to keep track on #431